jenkins/open-mpi.dist.create-tarball.groovy

```diff
@@ -109,9 +109,15 @@ parallel (
     node(manpage_builder) {
       stage('Build Man Pages') {
 	checkout_code();
-	sh "ls -lR . ; /bin/bash ompi-scripts/jenkins/open-mpi.dist.create-tarball.build-manpages.sh ${build_prefix} ${tarball} ${branch}"
+	sh """
+if test ! -f docs/requirements.txt; then
```

Comment:
Maybe a comment about why we do this?  Just to confirm, we're going to use read the docs for the official docs, right?